src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.hpp

```diff
@@ -1052,6 +1052,35 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_WriteThreadNetw
 
     namespace ThreadNetworkDiagnostics = app::Clusters::ThreadNetworkDiagnostics;
 
+    if (otDatasetIsCommissioned(mOTInst))
```

Comment:
Is this check backwards?
```suggestion
    if (!otDatasetIsCommissioned(mOTInst))
```

Reply:
Oh my, yes totally. I invert the condition to test the null encoded response case and forgot to switch it back when commiting